### PR TITLE
Add dev.omerkurt.Tureng

### PR DESCRIPTION
**Pull request by omerkurtdev**
### Please confirm your submission meets all the criteria

- [X] Please describe your application briefly.
-  A Turkish-English dictionary where all meanings of words, terms, idioms, phrases and usage are easily accessible.
- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I have [built][build] and tested the submission locally.
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

### CODE REVIEW

**Comment by omerkurtdev** [comment]
bot, build dev.omerkurt.Tureng

File: dev.omerkurt.Tureng.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{
+    "app-id" : "dev.omerkurt.Tureng",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "45",
```
Comment (hfiguiere) [inline dev.omerkurt.Tureng.json:4]: 46 is out

File: dev.omerkurt.Tureng.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+    "python3-modules.json",
+                {
+         "name":"blueprint",
```
Comment (hfiguiere) [inline dev.omerkurt.Tureng.json:30]: missing is the cleanup rule to remove blueprint after the build.

**Review by hfiguiere — CHANGES_REQUESTED** [review]
identation of the JSON is all over the place making it harder to review.

File: dev.omerkurt.Tureng.json [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+    "python3-modules.json",
+                {
```
Comment (bbhtt) [inline dev.omerkurt.Tureng.json:29]: Indentation is broken here.

File: dev.omerkurt.Tureng.json [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+         "buildsystem":"meson",
+         "sources":[
+            {
+               "type":"git",
+               "url":"https://gitlab.gnome.org/jwestman/blueprint-compiler",
+               "commit":"2a39a16391122af2f3d812e478c1c1398c98b972"
+            }
+         ]
+      },
+        {
```
Comment (bbhtt) [inline dev.omerkurt.Tureng.json:40]: Here too

File: dev.omerkurt.Tureng.json [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+            "name" : "Tureng",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+
+                    "type" : "git",
+           	    "tag": "v1.0.0",
```
Comment (bbhtt) [inline dev.omerkurt.Tureng.json:48]:
```suggestion
                    "type" : "git",
           	        "tag": "v1.0.0",
```

File: dev.omerkurt.Tureng.json [hunk 6]
```diff
@@ -0,0 +1,55 @@
+{
+  "app-id": "dev.omerkurt.Tureng",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "46",
```
Comment (omerkurtdev) [inline dev.omerkurt.Tureng.json:4]: I changed the runtime-version to 46.

File: dev.omerkurt.Tureng.json [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+    "/share/pkgconfig",
+    "*.la",
+    "*.a"
+  ],
+  "modules": [
+    "python3-modules.json",
+    {
+      "name": "blueprint",
+      "cleanup": [
+        "*"
```
Comment (omerkurtdev) [inline dev.omerkurt.Tureng.json:31]: added cleanup

**Comment by omerkurtdev** [comment]
> Your id in metainfo file does not match Flatpak ID https://github.com/omerkurtdev/dev.omerkurt.Tureng/blob/b7f92b0237bd1fba276b46faeda25ab07b29b67f/data/dev.omerkurt.Tureng.metainfo.xml.in#L3 has an extra `.desktop`.

https://github.com/omerkurtdev/dev.omerkurt.Tureng/blob/a40873a5c1cfc38ed12a55e1f0413e625571bb1d/data/dev.omerkurt.Tureng.metainfo.xml.in#L3C6-L3C25 
``.desktop``` extension removed

**Comment by hfiguiere** [comment]
bot, build dev.omerkurt.Tureng

**Comment by omerkurtdev** [comment]
> Is it expected that the window is not resizeable? It opens with some odd dimensions.
> 
>  Screencast.webm

Sorry, I missed it, I'll edit it and commit it again

**Comment by omerkurtdev** [comment]
bot, build dev.omerkurt.Tureng

**Comment by bbhtt** [comment]
Was the issue in https://github.com/flathub/flathub/pull/5169#issuecomment-2060113723 addressed? I don't see any new commits made here to update the version of the application?

File: dev.omerkurt.Tureng.json [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+    },
+    {
+      "name": "Tureng",
+      "builddir": true,
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "git",
+          "tag": "v1.0.0",
+          "url": "https://github.com/omerkurtdev/dev.omerkurt.Tureng.git"
```
Comment (hfiguiere) [inline dev.omerkurt.Tureng.json:50]: Retagging is bad.

please add the `commit` as well to protect against it.

Comment (omerkurtdev) [reply dev.omerkurt.Tureng.json:50]: I'm having a problem with the application icon, I have a few more things to edit, I'm sorry, I won't tag it again anymore, I'll add a direct commit.

https://github.com/omerkurtdev/dev.omerkurt.Tureng/commit/7c212b9e5f8b3749086e2fdb3db0ea74a2ef8cf1#diff-9790b2ef4d3a8a0757749534f76413e015621d1ad3bac24261f7d33e89785b57L5

**Comment by bbhtt** [comment]
bot, build dev.omerkurt.Tureng

**Pull request closed without merging**